chemreg/compound/tests/test_serializers.py

```diff
@@ -20,11 +20,24 @@ def test_invalid_inchikey(compound, invalid_inchikey):
     assert not serializer(data=json).is_valid()
 
 
-@pytest.mark.django_db
-def test_query_structure_type(querystructuretype):
+@pytest.mark.parametrize("compound", ["IllDefinedCompound"], indirect=["compound"])
+def test_ill_Defined_compound(compound, mrvfile):
+    """Test that an ill-defined compound can be created with the provided mrvfile."""
+    serializer = compound["serializer"]
+    json_factory = compound["json_factory"]
+    json = json_factory.build()
+    json["mrvfile"] = mrvfile
+    print("Validating:")
+    print(serializer(data=json).is_valid())
+    print(serializer(data=json).__dict__)
+    assert serializer(data=json).is_valid()
+
+
+@pytest.mark.django_db(transaction=True)
```

Comment:
```suggestion
@pytest.mark.django_db
```
We _**do**_ need database calls here, but we _**don't**_ need transaction support for these tests. This is similar to the difference between Django's `TestCase` and Django's `TransactionTestCase`... the former is a lot faster, but the latter is needed in some scenarios.